ruby/webapp.rb.in

```diff
@@ -1,5 +1,20 @@
 plugin_path = "@CMAKE_SOURCE_DIR@/public" 
 
+#load dirs containing scripts
```

Comment:
Oh god.

Just stumbled upon that ...

Either the package is a cmake package and it should be installed. Whole. Or a ruby package and you should not use cmake *at all* (especially since it is really not needed here). Using configure_file to inject the path to the current file is a very convoluted way to achieve this goal

__FILE__ gives you the path to the current file.